Directory.Build.props

```diff
@@ -7,10 +7,8 @@
     -->
 
   <PropertyGroup>
-    <ReactNativeWindowsDir>$(MSBuildThisFileDirectory)\vnext\</ReactNativeWindowsDir>
-    <ReactNativePackageDir>$(MSBuildThisFileDirectory)\node_modules\react-native\</ReactNativePackageDir>
-    <RootIntDir>$(ReactNativeWindowsDir)\build</RootIntDir>
-    <RootOutDir>$(ReactNativeWindowsDir)\target</RootOutDir>
+    <RootIntDir>$(MSBuildThisFileDirectory)vnext\build</RootIntDir>
+    <RootOutDir>$(MSBuildThisFileDirectory)vnext\target</RootOutDir>
```

Comment:
I see directory.build.props in vnext defines this, so this one will overwrite it? should these be set here only if they're no already set?